app/models/outcome.rb

```diff
@@ -14,17 +16,18 @@ class Outcome < ActiveRecord::Base
     reject_if: ->(attributes) { attributes[:level].blank? },
     allow_destroy: true
 
-  validates :name, presence: true, uniqueness: { scope: :course_id }
+  validates :label, presence: true, uniqueness: { scope: :course_id },
+    length: { maximum: 500 }
```

Comment:
Should the maximum be 5 characters? The schema reads `t.string   "label", limit: 5, null: false`.